src/doc/ndarray_for_numpy_users/mod.rs

```diff
@@ -116,6 +117,17 @@
 //! </tr>
 //! </table>
 //!
+//! # The ndarray ecosystem
+//!
+//! `ndarray` does not provide advanced linear algebra routines out of the box (e.g. SVD decomposition).
+//! Most of the routines that you can find in `SciPy.linalg`/`NumPy.linalg` are provided by another crate,
+//! [`ndarray-linalg`](https://crates.io/crates/ndarray-linalg).
+//!
+//! The same holds for statistics: `ndarray` provides some basic functionalities (e.g. `mean`)
+//! but more advanced routines can be found in [`ndarray-stats`](https://crates.io/crates/ndarray-stats).
+//!
+//! If you are looking to generate random arrays instead, check out [`ndarray-rand`](https://crates.io/crates/ndarray-rand).
+//!
```

Comment:
It may be worth mentioning `ndarray-npy` here for users who want to be able to read/write files compatible with NumPy.

Reply:
Definitely worth it, considering the audience we are talking to. Added it :+1: